test/source-sink-errors/mzcompose.py

```diff
@@ -207,11 +207,11 @@ def assert_error(self, c: Composition, error: str) -> None:
                 # Sinks generally halt after receiving an error, which means that they may alternate
                 # between `stalled` and `starting`. Instead of relying on the current status, we
                 # check that there is a stalled status with the expected error.
-                > SELECT bool_or(error ~* '{error}')
+                > SELECT bool_or(error ~* '{error}'), bool_or(details->'namespaced'->>'kafka' ~* '{error}')
```

Comment:
Check the namespaced error here https://github.com/MaterializeInc/materialize/blob/254e2c59f4bc9284399a2c42b7120fb9f58c5939/test/source-sink-errors/mzcompose.py#L114 as well?